src/apps/Hosting/HostingDialog.js

```diff
@@ -50,7 +50,11 @@ const CreateHostingDialog = React.createClass({
 
   handleAddSubmit() {
     const params = this.getHostingParams();
```

Comment:
put this line after state destructuring. let's keep that order when it's possible.